### PR TITLE
feat: 🎸 Action button enhancement for different size class

### PR DESCRIPTION
**Pull request by ShadowTourist**
Action button's max width is increased to 200 for landscape.

### CODE REVIEW

**Comment by justinguo** [comment]
Thanks for the PR. A minor suggestion for commit message:
- Bug fixes: usually begins with `fix: 🐛`
- New features or enhancements: `feat: 🎸`
- Document changes or refactor: e.g. `doc: 💡 ` 
Other than these, the PR looks good to me. Did you double confirm with `demo 8`? How this looks like on different devices, like iPhone 12 VS iPhone 12 Pro Max? I will also double check on my side.

**Review by MarcoEidinger — CHANGES_REQUESTED** [review]
@ShadowTourist please add/enhance SwiftUI preview implementation to allow quick tests for portrait and landscape mode. [Example](https://gist.github.com/MarcoEidinger/987f305aeb54170d8c7bc166637cb06f)

Also, it is helpful to drop screenshots into PR description/comment illustrating the before and after effect

|Before|After|
|---|---|
|beforeChange|afterChange|

I see the improvement in landscape mode for "List icon v..." button :)

But as you maybe can see the side effect is that "Footer button" label size also decreased which looks funky as the "View More" label size is different

**Comment by ShadowTourist** [comment]
@justinguo  Got it, I will update commit message later and I did't change size for Demo 8 which is a `Quick Replay` message. I will add width limitation for it too you can check this pr later I will also attach some preview for these update.

@MarcoEidinger 
Ok I will make preview for it. And as all components separated everywhere, bad for maintenance.

**Comment by justinguo** [comment]
@ShadowTourist Agree. There are places for improvements and we might discuss how to refactor them at a later point. Currently we hope to close the UI gaps between design and actual SDK ASAP, then we need to finish the research topics, at the end if have time we can pick some areas to improve.

Please address what Marco suggested first. We can have a quick sync-up tonight.

**Comment by ShadowTourist** [comment]
|portrait|landscape|
|--|--|
|Screen Shot 2021-08-06 at 2 37 57 PM|Screen Shot 2021-08-06 at 2 37 20 PM|

|Demo10|Demo8|
|--|--|
|Screen Shot 2021-08-06 at 2 24 27 PM|Screen Shot 2021-08-06 at 2 25 18 PM|

File: Sources/SAPCAI/UI/MessageView/QuickRepliesMessageView.swift [hunk 1]
```diff
@@ -18,20 +23,26 @@ struct QuickRepliesMessageView: View {
         assertionFailure("only use `QuickRepliesMessageView` with message type `quickReplies`")
         return nil
     }
+
+    var amountOfButtons: Int {
+        let limitForVisibleButtons = 11
```
Comment (justinguo) [inline Sources/SAPCAI/UI/MessageView/QuickRepliesMessageView.swift:28]: So in this case we are limiting number of quick reply buttons to 11?

**Comment by justinguo** [comment]
Please also address the conflict.

File: Sources/SAPCAI/UI/MessageView/QuickRepliesMessageView.swift (hunk 1 above)
Comment (MarcoEidinger) [reply Sources/SAPCAI/UI/MessageView/QuickRepliesMessageView.swift:28]: @justinguo yes, I introduced this in a different commit to align behavior with WebClient

Comment (justinguo) [reply Sources/SAPCAI/UI/MessageView/QuickRepliesMessageView.swift:28]: @MarcoEidinger Got it! So the solution looks good to me.

**Comment by MarcoEidinger** [comment]
@ShadowTourist I pushed a new commit to dev branch to ensure that header in files get striped when running `make format`. Please rebase, run `make format` and push.

P.S.: I strongly recommend to rebase (instead of merge) when pulling latest changes from dev branch into your local feature branch. This will avoid that other commits (not from you) are listed in your PR. If you have any questions about this I can explain how to do that in our next sync meeting

**Comment by ShadowTourist** [comment]
@MarcoEidinger  I prefer rebase too. I'm not familiar with fork work flow. I will check commits again. 
Pull request shows conflict, but I rebase dev (synced with sap:dev), all green, I will handle this, or I will resolve this on web.

**Comment by ShadowTourist** [comment]
Hi @MarcoEidinger  I dropped the merge commit and rebase the latest dev.

**Review by MarcoEidinger — APPROVED** [review]

**Pull request merged**